### DISCUSSION TITLE
Cloud providers platform

### DISCUSSION THREAD

**Discussion by nferrari** [gogs/gogs#6774] [Q&A] [answered]
Hi!

Would it be interesting to list the cloud providers having a documentation about Gogs, or even a fast deployment process?
As an example, see this [Gitea documentation page](https://docs.gitea.io/en-us/install-on-cloud-provider/
).

I'm ready to open a PR in that direction.

For information, we (alwaysdata) have listed Gogs on our [Marketplace](https://www.alwaysdata.com/en/marketplace/gogs/).

Regards,

**Comment by unknwon** [comment, marked as answer]
Hey @nferrari, thanks for asking!

It is always welcome, and I think you can add another item to the https://github.com/gogs/gogs#deploy-to-cloud section for alwaysdata.

**Comment by nferrari** [comment]
Alright thank you, I didn't pay attention to this section on the README.
PR incoming.

Regards,
